### PR TITLE
Increase `maxBuffer` to 10MB (1024*1024*10)

### LINKED ISSUE #8: RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stderr maxBuffer length exceeded
**Env:**

```
node -v
v10.2.1
npm -v
6.1.0
```

I've tried to install-local for large repository, I met following error.

```
[install-local] packing - 0/1{ RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stderr maxBuffer length exceeded
    at Socket.onChildStderr (child_process.js:362:14)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:279:12)
    at readableAddChunk (_stream_readable.js:260:13)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onread (net.js:636:20)
  cmd:
   'npm pack /my/large/repo' }
```

- [Errors | Node.js v10.4.1 Documentation](https://nodejs.org/api/errors.html#errors_err_child_process_stdio_maxbuffer)

` child_process.exec`'s `maxBuffer` is `200 * 1024` by default.

- [Child Process | Node.js v10.4.1 Documentation](https://nodejs.org/api/child_process.html#child_process_maxbuffer_and_unicode)

Can I avoid this error?

### PR DESCRIPTION
**Pull request by azu**
fix #8 

In my local, following tests is failed(master branch too)

- mac
- npm 6.1.0
- node.js 10.0.0

```
  36 passing (12s)
  1 failing

  1) install-local cli given 3 packages
       should install 2 packages and update the package.json if -S is provided:

      AssertionError: expected { Object (name, version, ...) } to deeply equal { Object (localDependencies, name, ...) }
      + expected - actual

       {
         "localDependencies": {
      -    "three": "../../../../../../../../var/folders/3h/ksxf_vgd08z_jy__g662s1rm0000gn/T/local-installer-it/three"
      -    "two": "../../../../../../../../var/folders/3h/ksxf_vgd08z_jy__g662s1rm0000gn/T/local-installer-it/two"
      +    "three": "../three"
      +    "two": "../two"
         }
         "name": "one"
         "version": "0.0.0"
       }

```

### CODE REVIEW

File: test/unit/LocalInstallerSpec.ts [hunk 1]
```diff
@@ -33,17 +33,19 @@ describe('LocalInstaller install', () => {
...
-            expect(execStub).calledWith(`npm pack ${resolve('/e')}`, { cwd: os.tmpdir() });
+            const maxBuffer = 1024 * 1024 * 10;
+            expect(execStub).calledWith(`npm pack ${resolve('b')}`, { cwd: os.tmpdir(), maxBuffer });
+            expect(execStub).calledWith(`npm pack ${resolve('c')}`, { cwd: os.tmpdir(), maxBuffer });
+            expect(execStub).calledWith(`npm pack ${resolve('/e')}`, { cwd: os.tmpdir(), maxBuffer });
         });
 
         it('should install correct packages', async () => {
             await sut.install();
+            const maxBuffer = 1024 * 1024 * 10;
```
Comment (nicojs) [inline test/unit/LocalInstallerSpec.ts]: Could we move this to the top of the test file? You may make it ALL_CAPS as well (like constant variables in Java)

File: src/LocalInstaller.ts [hunk 2]
```diff
@@ -71,7 +71,10 @@ export class LocalInstaller extends EventEmitter {
 
     private installOne(target: InstallTarget): Promise<void> {
         const toInstall = target.sources.map(source => resolvePackFile(source.packageJson)).join(' ');
-        const options: ExecOptions = { cwd: target.directory };
+        const options: ExecOptions = {
+            cwd: target.directory,
+            maxBuffer: 1024 * 1024 * 10
```
Comment (nicojs) [inline src/LocalInstaller.ts]: Could we replace this with a constant? 

```ts
const TEN_MEGA_BYTE = 1024 * 1024 * 10;
```

Can be placed at the top of the file and reused where needed.

**Comment by nicojs** [comment]
@azu do you have the time to do the changes? Or shall I do them?

**Comment by azu** [comment]
Sorry, too late.
I've fixed it.

**Comment by nicojs** [comment]
Awesome! Thanks :handshake: 

I'll create a new release after #11 is merged

**Pull request merged**